src/steamship/invocable/mixins/__init__.py

```diff
@@ -0,0 +1,15 @@
+from enum import Enum
+from typing import Any, Dict
+
+from steamship import Tag
+
+
+class FileType(str, Enum):
+    YOUTUBE = "YOUTUBE"
+    PDF = "PDF"
+    WEB = "WEB"
+    TEXT = "TEXT"
+
```

Comment:
(continuing our in person chat) -- I think relying on the existing MimeType we've got is probably simplifying here.

PDF -> MimeTypes.PDF
WEB -> MimeTypes.HTML
TEXT -> MimeTypes.TXT
YOUTUBE You could have a few options and then build in the special casing just as you'd already done. E.g. could extend the MimeTypes class to include things like:

```
    YOUTUBE_TRANSCRIPT = "application/youtube-transcript"
    YOUTUBE_VIDEO = "application/youtube-video"
```